services/core-api/app/api/now_submissions/response_models.py

```diff
@@ -407,6 +436,9 @@
         'under_exp_surface_activity': fields.List(fields.Nested(UNDER_EXP_SURFACE_ACTIVITY)),
         'water_source_activity': fields.List(fields.Nested(WATER_SOURCE_ACTIVITY)),
         'mech_trenching_activity': fields.List(fields.Nested(MECH_TRENCHING_ACTIVITY)),
+        'camps': fields.List(fields.Nested(CAMP_ACTIVITY)),
+        'buildings': fields.List(fields.Nested(BUILDING_ACTIVITY)),
+        'stagingareas': fields.List(fields.Nested(STAGING_AREA_ACTIVITY)),
```

Comment:
Should this be "staging_areas" and not "stagingareas"?

Reply:
vfcbc is sending the key as stagingareas, so I've named everything the same.